src/main/webapp/js/omniComment.js

```diff
@@ -220,7 +220,7 @@ $(document).ready(function() {
         
         // to show student comments (only works for Giver filter)
         if (commentToShow.hasClass('student-record-comments')) {
-            var studentCommentPanelBody = commentToShow.closest('.student-comments-panel').parent();
+            var studentCommentPanelBody = commentToShow.parent();
```

Comment:
Just tested, it appears that there is no change in behaviour, since disabling the view for all "status_display\*" comments does not hide the student comments panel. Adopting this change.